src/main.rs

```diff
@@ -67,24 +69,29 @@ fn main() {
 
     if attrs.is_empty() {
         eprintln!("no match");
-        std::process::exit(1)
+        return ExitCode::FAILURE;
     }
 
-    let attrs: Vec<&str> = std::str::from_utf8(&attrs)
+    let attrs: Vec<_> = std::str::from_utf8(&attrs)
         .expect("fail")
         .trim()
         .split('\n')
         .collect();
 
-    let choice = if attrs.len() != 1 {
-        pick(&args.picker, attrs)
+    dbg!(&attrs);
```

Comment:
leftover from debugging?

Reply:
if you'd like you can use some log crate and a `--verbose` option

Reply:
Whoops, yeah i left that in by accident. I'll remove it.